### DISCUSSION TITLE
Releasing Apps that are approved in Google Play console

### DISCUSSION THREAD

**Discussion by eamonn-alphin** [fastlane/fastlane#17979] [Q&A]
In IOS, I can upload the app in one step, submit it for review in another, and then release it in a third step. 

In Google, I have to upload and submit in the same step, then (if I have managed publishing turned on), once the app is approved, I have to log in, and manually publish it. Is there any way to set up a lane that can automatically release apps for me, when I run it?

**Comment by LucasPMM** [comment]
Hello, I'm facing the same situation. I would like to know if there is a way to release android apps when `Managed publishing` is turned on too.

Another point is if it's possible to enable/disable android `Managed publishing` using fastlane, such as the iOS implementation using the param `distribute_only:"true || false"` on `fastlane run pilot` command

**Comment by LucasPMM** [comment]
Hello guys, any response?

**Comment by raldred** [comment]
There doesn't seem to be any specific support for this in the Android Publisher API V3 so I don't think it will be supported.
Unless, with managed publishing enabled you can set `rollout` to 0 and `release_status` to completed, when it's done in review, you can set `rollout` to 1

**Reply by nbitzer** [reply]
@phillco ^

**Reply by LucasPMM** [reply]
<img width="585" alt="Captura de Tela 2021-08-02 às 13 34 41" src="https://user-images.githubusercontent.com/23710304/127894570-c404b1b3-6006-4eea-8adf-c0538e4480d0.png">

**Reply by raldred** [reply]
Well that's out then. Shame.

**Reply by oiesc** [reply]
Set rollout value to 0.0000001, it will work.

**Comment by Fabiolagg** [comment]
I have heard that this year the approval time has been increased to 7 days or more. I've also read that if your app is 18+ then the approval process is faster?

I have published apps prior to this and it took a few hours, but this was a few years ago. I am building a new google developer account for my new startup and we need to publish the app within a week.

Can someone please advise?


[rapidfs](https://www.rapidfs.me/)

**Reply by raldred** [reply]
That's something you'd have to ask Google. This isn't the place for general play store publishing questions, sorry.

**Comment by akoueikjc** [comment]
hi guys,
does anyone found a solution for this ?

**Comment by akashchauhan0855** [comment]
Hello guys, anyone found any solution for this ?

**Reply by raldred** [reply]
As far as I'm aware, there is still no capability for this in Android Publisher API

**Reply by oiesc** [reply]
I upload my apps with the smallest possible rollout (0.0000001), and then set it to another value when I want my users to have access (Managed publishing off).
When I check the total number of devices available for this rollout value, the store shows 0.
<img width="547" height="235" alt="Screenshot 2026-04-20 at 16 18 35" src="https://github.com/user-attachments/assets/52f1c58b-1c64-4831-9667-56eaa3a22b8e" />
